### PR TITLE
Fix tests relying on push-error and push-message IDs on modals. (no JIRA)

### PR DESCRIPTION
**Pull request by breilly2**
This required a little bit of adding data-test-ids and a little bit of changing tests to expect a message modal instead of an error modal. I'm asking both @rushtong and @MattPutnam for review to make sure that change from error -> message is intentional and I'm not sure who made that change.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by rushtong — APPROVED** [review]
👍  LGTM

File: src/cljs/main/broadfcui/components/modals.cljs [hunk 1]
```diff
@@ -83,7 +83,8 @@
 
 (defn render-error [{:keys [header text on-dismiss icon-color]}]
   [OKCancelForm
-   {:header [:div {:style {:display "inline-flex" :align-items "center"}}
+   {:data-test-id "push-error"
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/components/modals.cljs]: "push-error" seems like a stale name

File: src/cljs/main/broadfcui/components/modals.cljs [hunk 2]
```diff
@@ -93,7 +94,8 @@
 
 (defn render-message [{:keys [header text on-confirm on-dismiss]}]
   [OKCancelForm
-   {:header (or header "Confirm")
+   {:data-test-id "push-message"
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/components/modals.cljs]: same

**Review by MattPutnam — APPROVED** [review]
👍

File: src/cljs/main/broadfcui/components/modals.cljs (hunk 2 above)
Comment (breilly2) [reply src/cljs/main/broadfcui/components/modals.cljs]: @MattPutnam, I wasn't too fond of them either. I was just recovering what had been there. Would `error-modal` and `message-modal` work for you? If not, please suggest something better.

**Pull request merged**